kornia/geometry/boxes.py

```diff
@@ -164,6 +157,9 @@ def _boxes3d_to_polygons3d(
     return polygons3d
 
 
+T = TypeVar("T", bound="Boxes")
```

Comment:
Why?

Reply:
For the child class methods to return child class type